pkg/cmd/verifyEnv.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/verify"
+)
+
+// verifyEnvCmd represents the verifyEnv command
+var verifyEnvCmd = &cobra.Command{
+	Use:   "verifyEnv",
```

Comment:
Can you make this `env verify` or just `verify`? I foresee cases of wanting other subcommands on an `env` command e.g. `env dump` or `env findInterface` if it's a lot of work we can do it later.